.github/workflows/gradle-build.yml

```diff
@@ -19,6 +19,6 @@ jobs:
         java-version: '8'
         distribution: 'temurin'
     - name: Set up Gradle Build Action
-      uses: gradle/gradle-build-action@v2.2.1
```

Comment:
btw, should we update this line to just `gradle/gradle-build-action@v2`?

Reply:
Yes that would be great.

Reply:
Implemented!